viewer/vueapp/src/components/settings/Settings.vue

```diff
@@ -1507,6 +1507,10 @@ export default {
     },
     notifiers () {
       return this.$store.state.notifiers;
+    },
+    disablePassword () {
```

Comment:
Because Cont3xt uses vuex store getters to get the user (see `getUser` in the `mapGetters` above) so when the `disablePassword` computed property is initialized, the `getUser` getter has not been initialized yet.

Viewer is old school and doesn't use these getters, so user user is defined when the `disablePassword` computed property is initialized!